src/bundle/Resources/public/js/scripts/fieldType/ezdate.js

```diff
@@ -94,8 +94,9 @@
 
         if (sourceInput.value) {
             defaultDate = new Date(sourceInput.value * 1000);
+            const timezoneOffset = sourceInput.dataset.timezone_offset ? sourceInput.dataset.timezone_offset : defaultDate.getTimezoneOffset();
 
-            defaultDate.setTime(defaultDate.getTime() + defaultDate.getTimezoneOffset() * 60 * 1000);
```

Comment:
Why did you removed `* 60`?

Reply:
Resolved in https://github.com/ezsystems/ezplatform-admin-ui/pull/1185/commits/2af3520ee22180bf3f5f8d5d4a4d75f4b39b56c0